pkg/sql/schema_adapter_test.go

```diff
@@ -1,19 +1,83 @@
 package sql_test
 
 import (
+	"context"
 	"strings"
+	"testing"
+	"time"
 
+	"github.com/stretchr/testify/require"
+
+	"github.com/ThreeDotsLabs/watermill"
 	"github.com/ThreeDotsLabs/watermill-sql/pkg/sql"
+	"github.com/ThreeDotsLabs/watermill/message"
 )
 
-// testSchema makes the following changes to DefaultSchema to comply with tests:
+func TestDefaultMySQLSchema(t *testing.T) {
```

Comment:
it would be nice to have some comment, explaining what this check is checking and why ;)

Reply:
just to not forget

Reply:
See cc53b9e2bd237d45bd0fd44441b787a6a311dcab